src/frontend/src/pages/Index/Settings/AccountSettings/UserThemePanel.tsx

```diff
@@ -26,50 +26,29 @@ const LOOKUP = Object.assign(
 );
 
 export function UserTheme({ height }: { height: number }) {
-  // primary color
+  const [themeLoader, setThemeLoader] = useLocalState((state) => [
+    state.loader,
+    state.setLoader
+  ]);
+
+  const theme = useMantineTheme();
+
+  // Set theme primary color
   function changePrimary(color: string) {
     useLocalState.setState({ primaryColor: LOOKUP[color] });
   }
-  // white color
-  const [whiteColor, setWhiteColor] = useState(theme.white);
-  function changeWhite(color: string) {
-    useLocalState.setState({ whiteColor: color });
-    setWhiteColor(color);
-  }
-  // black color
-  const [blackColor, setBlackColor] = useState(theme.black);
-  function changeBlack(color: string) {
-    useLocalState.setState({ blackColor: color });
-    setBlackColor(color);
-  }
-  // radius
-  function getMark(value: number) {
-    const obj = SizeMarks.find((mark) => mark.value === value);
-    if (obj) return obj;
-    return SizeMarks[0];
-  }
-  function getDefaultRadius() {
-    const obj = SizeMarks.find(
-      (mark) => mark.label === useLocalState.getState().radius
-    );
-    if (obj) return obj.value;
-    return 50;
-  }
-  const [radius, setRadius] = useState(getDefaultRadius());
-  function changeRadius(value: number) {
```

Comment:
Do I get this right, that this removes the ability to use a "rounder" radius than by default? I really liked to set it to a rounder radius than default, so this was really helpful. I once thought about defining default options on the via the config/env for these values here, because I always have to change this on a new client, and I know that pushing a rounder radius to core is not well liked by others.